lib/util/http/github.ts

```diff
@@ -273,12 +278,33 @@ export class GithubHttp extends Http<GithubHttpOptions, GithubHttpOptions> {
     options?: InternalHttpOptions & GithubHttpOptions,
     okToRetry = true
   ): Promise<HttpResponse<T>> {
-    const opts = {
+    const opts: GithubHttpOptions = {
       baseUrl,
       ...options,
       throwHttpErrors: true,
     };
 
+    if (!opts.token) {
+      const authUrl = new URL(resolveBaseUrl(opts.baseUrl!, url));
+
+      if (opts.repository) {
```

Comment:
Is this case only when GraphQL is used?

Reply:
currently yes. in other cases we already have the right url